src/include/net_connect.h

```diff
@@ -88,37 +88,27 @@ typedef unsigned long pbs_net_t;        /* for holding host addresses */
 #define	IM_PROTOCOL_VER	6	/* inter-mom protocol version number */
 #define	IM_OLD_PROTOCOL_VER 5	/* inter-mom old protocol version number */
 
-#define	IS_PROTOCOL	4	/* inter-server protocol number */
-#define	IS_PROTOCOL_VER	3	/* inter-server protocol version number */
+#define	IS_PROTOCOL	5	/* inter-server protocol number */
```

Comment:
I don't get you. In that case, IS_PROTOCOL will never change? I incremented IS_PROTOCOL itself as this is a major change.

Reply:
Yes, ideally IS_PROTOCOL should not change as its just an identifier used by do_tpp() (and other related funcs)  to find which msg is received so it can call appreciate function to handle it. However IS_PROTOCOL_VER is what specifies the current version of IS protocol, and that's what we need to change.